src/js/components/AppPageComponent.jsx

```diff
@@ -182,6 +182,26 @@ var AppPageComponent = React.createClass({
     }
   },
 
+  getUnhealthyTaskMessage: function (healthCheckResults = []) {
+    return healthCheckResults.map((hc, index) => {
+      if (hc && !hc.alive) {
+        var failedCheck = this.state.app.healthChecks[index];
+
+        var protocol = failedCheck.protocol
+          ? `${failedCheck.protocol} `
+          : "";
```

Comment:
As I said I'm all for coming to an agreement and actually **enforcing** the result of our discussion, whatever it may be. 
For the moment, though, keep in mind that these ternaries were already there – all I did was introduce the template strings. 

Also, ternaries are widely in the codebase, and I'm of the opinion that it makes sense to refactor them at once after we come to an agreement.